components/datadog/apps/fakeintake/fargateFakeintakeService.go

```diff
@@ -122,7 +112,45 @@ func fargateLinuxContainerDefinition(imageURL string) *ecs.TaskDefinitionContain
 				Protocol:      pulumi.StringPtr("tcp"),
 			},
 		},
-		VolumesFrom: ecs.TaskDefinitionVolumeFromArray{},
+		HealthCheck: &ecs.TaskDefinitionHealthCheckArgs{
+			Command: pulumi.ToStringArray([]string{"CMD-SHELL", "curl --fail http://localhost/fakeintake/health"}),
+		},
+		DockerLabels: pulumi.StringMap{
+			"com.datadoghq.ad.checks": pulumi.String(utils.JSONMustMarshal(
+				map[string]interface{}{
+					"openmetrics": map[string]interface{}{
+						"init_config": map[string]interface{}{},
+						"instances": []map[string]interface{}{
+							{
+								"openmetrics_endpoint": "http://%%host%%/metrics",
+								"namespace":            "fakeintake",
+								"metrics": []string{
+									".*",
+								},
+							},
+						},
+					},
+					"http_check": map[string]interface{}{
+						"init_config": map[string]interface{}{},
+						"instances": []map[string]interface{}{
+							{
+								"name": "health",
+								"url":  "http://%%host%%/fakeintake/health",
+							},
+							{
+								"name": "metrics query",
+								"url":  "http://%%host%%/fakeintake/payloads?endpoint=/api/v2/series",
+							},
+							{
+								"name": "logs query",
+								"url":  "http://%%host%%/fakeintake/payloads?endpoint=/api/v2/logs",
+							},
+						},
+					},
+				}),
+			),
+		},
+		LogConfiguration: ecsClient.GetFirelensLogConfiguration(pulumi.String("fakeintake"), pulumi.String("fakeintake"), apiKeySSMParamName),
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
🤩